blog/tripleo.md

```diff
@@ -1,13 +1,252 @@
+
+# copy Skydive resources to the undercloud
+scp -F ~/.quickstart/ssh.config.ansible -r skydive.git undercloud:
+scp -F ~/.quickstart/ssh.config.ansible skydive undercloud:
+
+# make Skydive ansible available for mistral 
+ssh -F ~/.quickstart/ssh.config.ansible undercloud "sudo cp -R /home/stack/skydive.git/contrib/ansible /usr/share/ansible/skydive-ansible"
+
+# copy Tripleo skydive environment file
+scp -F ~/.quickstart/ssh.config.ansible $SKYDIVE_CONFIG undercloud:skydive.yaml
+
+# prepare the overcloud deployment
+pushd $QUICKSTART
+bash quickstart.sh -R master --no-clone --tags all \
+	--config $CONFIG \
+	-I --teardown none -p quickstart-extras-overcloud-prep.yml $VHOST
+
+# rebuild kolla images and update the environment file
+ssh -F ~/.quickstart/ssh.config.ansible undercloud <<'EOF'
+REGISTRY=$(grep push_destination containers-prepare-parameter.yaml | head -n 1 | awk '{print $3}' | tr -d '"' )
+
+sudo iptables -I INPUT -p tcp --dport 18888 -j ACCEPT
+python -m SimpleHTTPServer 18888 &
+HTTP_SERVER=$!
+
+ADDRESS=$(ifconfig docker0 | awk '/inet /{print $2}')
+
+rm -rf kolla
+git clone https://github.com/openstack/kolla
+
+pushd kolla
+sed -i "s|https://github.com/skydive-project/skydive/releases/download/\(.*\)/skydive|http://$ADDRESS:18888/skydive|" docker/skydive/skydive-base/Dockerfile.j2
+tools/build.py --registry $REGISTRY --push -b centos skydive-agent --tag devel --network_mode host --nocache
+tools/build.py --registry $REGISTRY --push -b centos skydive-analyzer --tag devel --network_mode host --nocache
+popd
+
+echo "Kolla docker images pushed"
+
+echo "  SkydiveAnsiblePlaybook: /usr/share/ansible/skydive-ansible/playbook.yml.sample" >> skydive.yaml
+
+echo "  DockerSkydiveAnalyzerImage: $REGISTRY/kolla/centos-binary-skydive-agent:devel" >> skydive.yaml
+echo "  DockerSkydiveAgentImage: $REGISTRY/kolla/centos-binary-skydive-agent:devel" >> skydive.yaml
+
+kill $HTTP_SERVER
+EOF
+
+# start the overcloud deployment
+bash quickstart.sh -R master --no-clone --tags all \
+	--config $CONFIG \
+	-I --teardown none -p quickstart-extras-overcloud.yml $VHOST
+popd
+{% endhighlight %}
+
+# First Skydive request
+
+Once deployed we can go to the undercloud machine.
+
+{% highlight shell %}
+ssh -F ~/.quickstart/ssh.config.ansible undercloud
+{% endhighlight %}
+
+There we need to get the controller IP where the Skydive analyzer runs.
+
+{% highlight shell %}
+source stackrc
+CTLIP=$( openstack server show overcloud-controller-0 -f json | jq -r .addresses | cut -d '=' -f 2 )
+{% endhighlight %}
+
+Finally let's do our first Skydive request. The following one will return all the OpenvSwitch bridges.
+
+{% highlight shell %}
+export SKYDIVE_ANALYZER=$CTLIP:8082
+./skydive client query "G.V().Has('Type', 'ovsbridge')" --username skydive --password secret
+{% endhighlight %}
+
+
+## Conclusion
+
+As we saw most of the job is there to use the latest version of Skydive with Tripleo. Once the remaining patches
```

Comment:
As we just saw, most of the job is there to build and push Kolla images using the latest version of Skydive.

Reply:
Once the remaining patches are merged and the 0.19 version of Skydive is released, providing the Skydive environment file (`environments/services/skydive-environment.yaml`) and adding